.github/CONTRIBUTING.md

```diff
@@ -39,7 +40,7 @@ As contributors you should be respectful and considerate of others - both contri
 
   To lint the code run at the command prompt.
   ```
-  npm run lint
+  npm run precommit
```

Comment:
@slvnperron 
the `lint` script has been removed from the package.json
what should replace that command in the documentation?

Reply:
@jsalwen We still have `npm run lint` which runs `eslint`